console/console-init/ui/src/modules/address-space/AddressSpacePage.tsx

```diff
@@ -100,12 +68,19 @@ export default function AddressSpacePage() {
 
   const onConfirmDeleteAll = async () => {
     if (selectedAddressSpaces && selectedAddressSpaces.length > 0) {
-      const data = selectedAddressSpaces;
-      await Promise.all(
-        data.map((addressSpace, index) =>
-          deleteAddressSpace(addressSpace, index)
-        )
+      let queryVariables: Array<{ name: string; namespace: string }> = [];
```

Comment:
I think, deleteAddressSpaceVariables makes more sense, wdyt?

Reply:
And similar changes in the other query variables as well.

Reply:
This is just a function scope variable and not a global scope variable. So i think queryVaraible is fine.